client/src/Pages/User/Orders/index.js

```diff
@@ -2,120 +2,271 @@ import React, { useState } from "react";
 import Modal from "react-modal";
 import { useQuery, gql, useMutation } from "@apollo/client";
 import { useHistory } from "react-router";
+import { Tab, Tabs, TabList, TabPanel } from "react-tabs";
+import "./tabs.css";
 import "./order.css";
 
 Modal.bind("#app");
 
 const GET_PAST_ORDERS = gql`
-	query GetOrders {
-		orderMany(
-			filter: {
-				OR: [
-					{ fulfillment: Ready }
-					{ fulfillment: Cancelled }
-					{ fulfillment: Placed }
-					{ fulfillment: Preparing }
-				]
-			}
-		) {
-			_id
-			__typename
-			vendor {
-				name
-			}
-			items {
-				product {
-					name
-				}
-			}
-			user {
-				netid
-			}
-			fulfillment
-		}
-	}
+  query GetOrders {
+    orderMany(
+      filter: {
+        OR: [
+          { fulfillment: Ready }
+          { fulfillment: Cancelled }
+          { fulfillment: Placed }
+          { fulfillment: Preparing }
+        ]
+      }
+    ) {
+      _id
+      __typename
+      vendor {
+        name
+      }
+      items {
+        product {
+          name
+          price
+        }
+      }
+      user {
+        netid
+      }
+      fulfillment
+      createdAt
+    }
+  }
 `;
 
 const CANCEL_ORDER = gql`
-	mutation CancelOrder($_id: MongoID!) {
-		orderUpdateOne(
-			filter: { _id: $_id }
-			record: { fulfillment: Cancelled }
-		) {
-			record {
-				_id
-				__typename
-				fulfillment
-			}
-		}
-	}
+  mutation CancelOrder($_id: MongoID!) {
+    orderUpdateOne(filter: { _id: $_id }, record: { fulfillment: Cancelled }) {
+      record {
+        _id
+        __typename
+        fulfillment
+      }
+    }
+  }
 `;
 
+function roundMoney(value) {
+  return Math.round((value + Number.EPSILON) * 100) / 100;
+}
+
 const OrderItemList = ({ items }) => {
-	return (
-		<div>
-			<p>Order Items: </p>
-			{items.map((item) => (
-				<p>{item.product.name}</p>
-			))}
-		</div>
-	);
+  return (
+    <div>
+      {items.map((item) => (
+        <span className="item">
+          <p>{item.product.name}</p>{" "}
+          <p>${roundMoney(parseFloat(item.product.price))}</p>
```

Comment:
We'll also want to format this to two decimal places.

Reply:
roundMoney() already formats the price to two decimal places. If you're talking about why it's not $0.00 for $0 priced items it shouldn't matter since it's a bug that an empty order can be placed.

Reply:
![Untitled](https://user-images.githubusercontent.com/19421319/88429284-31fd3180-cdab-11ea-96ea-c7ae4d38fc13.png)

Reply:
You can do something like `num.toFixed(2)` to format to two decimal places.

Reply:
Hm it rounds to 2 decimal places for me for tax and total. I used num.toFixed to fix the thai teas though. Let me know what you see in the next pr